websites/T/Trello/dist/metadata.json

```diff
@@ -14,9 +14,9 @@
     "blog.trello.com",
     "developers.trello.com"
   ],
-  "version": "2.1.9",
-  "logo": "https://i.imgur.com/5IK6wjd.png",
-  "thumbnail": "https://i.imgur.com/6V8VR6S.png",
+  "version": "2.1.91",
+  "logo": "https://i.imgur.com/KBKCgLY.png",
+  "thumbnail": "https://i.imgur.com/Jb3aSPJ.png",
```

Comment:
Logo is blurry please fix,
thumbnail is in dutch, its an English website so please take one in English.